devices/flash-flexspi/fspi/fspi.c

```diff
@@ -86,11 +81,41 @@ __attribute__((section(".noxip"))) void flexspi_lutUpdate(flexspi_t *fspi, u32 i
 }
 
 
+__attribute__((section(".noxip"))) void flexspi_lutUpdateEntries(flexspi_t *fspi, u32 index, const u32 *lutTable[], size_t elems, size_t count)
+{
+	u32 lutCopy[64];
+	const u32 *src;
+	unsigned int i, j;
+
+	for (i = 0; i < elems; ++i) {
+		src = *(lutTable++);
+		for (j = 0; j < count; ++j) {
+			lutCopy[i * count + j] = ((src != NULL) ? src[j] : 0);
```

Comment:
Why do we want to handle NULL in this manner?

Reply:
Pointer e.g. `[fspi_cmdCustomX] = NULL`, and as example`[fspi_enter4byteAddr]` is currently set for Micron only. If particular pointer is empty, LUT entry will be zeroed.